content/en/users/check-in/vos/_index.md

```diff
@@ -9,6 +9,23 @@ description: >
 This page contains information about using Check-in for managing your Virtual
 Organisation (VO).
 
+## Background
+
+In simple terms a Virtual Organisation (VO) is just a group of users. In
+EGI VOs are created to group researchers who aim to share resources across the
+EGI Federation to achieve a common goal as part of a scientific collaboration.
+
+You can browse existing VOs in the
+[EGI Operations Portal](https://operations-portal.egi.eu/vo/a/list). For each
+VO you can click on the _Details_ link to get more information. You can join
+an existing VO either using the _enrollment URL_ or emailing VO managers. See
+for example VOs for
+[WeNMR](https://operations-portal.egi.eu/vo/view/voname/enmr.eu) or
+[EMPHASIS](https://operations-portal.egi.eu/vo/view/voname/vo.emphasisproject.eu).
```

Comment:
I'm not sure we should point to user VO like those one, that may also disappear. If you want to give an example it may be better to use dteam or ops that are generic VO that will always stay there.

Reply:
Thanks, I didn't thought about that. I had a look at the `ops` and `dteam` VOs but I wanted to provide examples in a specific research domain to make it more representative. Therefore, I think it is better to remove examples. Providing https://operations-portal.egi.eu/vo/a/list I think is the most important part.

Reply:
Yes, I fully agree. All looks good to me, thanks!
Maybe someone for TST/CST will want to review this PR too.